src/lib/components/Tabs/index.js

```diff
@@ -271,10 +275,11 @@ export class Tabs extends BaseElement {
   }
 
   /**
-   * @param {!Node} node to check
+   * @param {HTMLElement} node to check
    * @return {number} the index of the tab containing this node, or -1 for none
    */
   indexOfTabByChild(node) {
+    /** @type HTMLDivElement */
```

Comment:
I don't think we care that this is a `div`, explicitly. Could just be `HTMLElement`?